### ISSUE TITLE
Crash when using Ctrl + MouseWheel to zoom in/out map in Windows x64

### ISSUE THREAD

**Issue by Chaoseiro** [mapeditor/tiled#782]
The title says it, Tiled just crashes when using Ctrl + MouseWheen to zoom in or out on a map. [Here's a map I was using when I noticed this crash](https://drive.google.com/file/d/0B94sVpuHml_tV1B0SVhOMWVoMEU/view?usp=sharing)

Some maps crashes right away, others take some zooming to crash.

Using Daily Build 2014-09-27 win32

**Comment by bjorn** [comment]
@ArmagonNo1 Maybe you would have a chance to run Tiled in a debugger to find out more information about this crash, as I requested in my last comment?

**Comment by ArmagonNo1** [comment]
I'm no programmer, but one of my co-workers will probably be able to take a look at it on Monday and I'll get back to you!

**Comment by toddcarnes** [comment]
For what it's worth, I compiled from the git repo and was unable to reproduce the error in 64 bit Windows 7.

**Comment by tcoxon** [comment]
@bjorn: @ArmagonNo1 and I had a go at getting a stack trace in Visual Studio. This was with the released binary v0.11.0-25-g521a40a on Windows 7 Version 6.1 (Build: 7601: Service Pack 1). It wasn't with a debug build, so we have no symbols or debug information unfortunately.

The scrollwheel crash occurred at 0x07ddfd0b. Stack frames below the top one were missing/corrupted, so I have no other information to give you about that.

We also encountered a crash when opening a large file, and we're not sure if that's related. The stack trace for that was:

```
0669fa72()  Unknown
  [Frames below may be incorrect and/or missing]  
  nvoglv32.dll!033b7e89() Unknown
  nvoglv32.dll!02b8497b() Unknown
  nvoglv32.dll!0313000a() Unknown
  nvoglv32.dll!02b006b6() Unknown
  nvoglv32.dll!032b789d() Unknown
  nvoglv32.dll!02ae2d91() Unknown
  nvoglv32.dll!02ae2fe3() Unknown
  nvoglv32.dll!02ae3a6c() Unknown
...
```

**Comment by bjorn** [comment]
Hmm, that stack trace is pretty much useless. It shows some NVidia OpenGL DLL but with the message above that it may be incorrect. Do you have OpenGL acceleration mode turned on in the Tiled preferences?

**Comment by tcoxon** [comment]
Yes, we tried turning off hardware acceleration in Tiled preferences a few minutes ago and the crashes don't occur any more.

**Comment by ArmagonNo1** [comment]
Upon further investigation, the issue appears to be graphics driver related, as disabling the hardware acceleration seems to have stopped the crashing. Prior to disabling the hardware acceleration these were the stacks I encountered:

> 066bfd0b()  Unknown
  [Frames below may be incorrect and/or missing] 

  > 07d5fd0b()  Unknown
  [Frames below may be incorrect and/or missing]  

For your reference I was running with these:
TILED v0.11.0-52-g129ef0f
Windows 7 Home Premium - Version 6.1 (Build 7601: Service Pack 1)
GeForce Game Ready Driver 344.75

**Comment by bjorn** [comment]
Ok, strack trace not so useless after all. :-)

Unfortunately, that does mean there is an instability outside of Tiled and I would have no idea how to avoid triggering that crash, short of disabling the OpenGL mode (which fortunately, is not at all that useful and disabled by default).

**Comment by tcoxon** [comment]
I think we're happy for now just disabling hardware acceleration. If we have any more relevant information, we'll let you know.

**Comment by ArmagonNo1** [comment]
Understandable, if it's outside of your control then it can't be helped. Disabling it solves my immediate problem so I'm quite happy to leave it off.
